tests/yast2_gui/yast2_firewall.pm

```diff
@@ -70,6 +70,18 @@ sub susefirewall2 {
     send_key "alt-f";
 }
 
+sub detect_dialog_flickering {
```

Comment:
I'm not sure about naming, what is actual purpose of the method, except that it's a workaround for instability of the test? Something like `assert_dialog_closed` sounds better to me. If it's about bug detection only, we could put bug reference in the name.  WDYT?